test/tcp.c

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
I am testing against a Pion server right now. I am going to do a little faux ice-tcp server in the same file to test against.